### PR TITLE
Cleanup repo as preperation for release

### PR DESCRIPTION
**Pull request by andrerom**
Now pointing to ezpublish-community and eZ Publish Platform 5.x for legacy needs,
same will be done in doc and everywhere to make it possible to learn platform
w/o known anything about legacy.

review ping @bdunogier @lolautruche

### CODE REVIEW

File: doc/nginx/etc/nginx/sites-available/mysite.com [hunk 1]
```diff
@@ -9,19 +9,13 @@ server {
...
     ## Don't forget to run php ezpublish/console assetic:dump --env=prod
     ## and make sure to comment these out in DEV environment.
     include ez_params.d/ez_prod_rewrite_params;
 
-    # ezlegacy cluster rewrite rules, uncomment if vhost uses DFS clustering
-    # For 5.4+:
-    #include ez_params.d/ez_cluster_rewrite_params;
-    # For 5.3:
-    #include ez_params.d/5.3_cluster/ez_cluster_rewrite_params;
+    # cluster rewrite rules, uncomment if vhost uses DFS clustering
```
Comment (andrerom) [inline doc/nginx/etc/nginx/sites-available/mysite.com]: open question: is this really just relevant to DFS? depends on the IO handler right?

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ requirements and best practices remain the same.
 The best way to kickstart is to read the [Symfony2 documentation](http://symfony.com/doc/current/book/page_creation.html)
 in order to get the basics.
 
+@TODO: Section below is out-of-date but represent some hints on how to be learn more about contributing to eZ Platform.
+       For introduction to *using* eZ Platform, please check our online doc.
```
Comment (lolautruche) [inline GETTING_STARTED.md]: A link to online doc would be nice :wink:

File: README.md [hunk 3]
```diff
@@ -58,11 +58,14 @@ Submitting bugs, improvements and stories is possible on https://jira.ez.no/brow
...
 
 ## Backwards compatibility
-eZ Platform is **100% data compatible** with eZ Publish 5.x, 4.x and 3.x *(introduced in 2002)*, meaning that content in those versions of the CMS can be upgraded using
- [online documentation](http://doc.ez.no/eZ-Publish/Upgrading) to eZ Platform.
-
-Unlike eZ Publish Platform 5, eZ Platform doesn't ship with eZ Publish Legacy (4.x). *Legacy* can however still be installed using Composer,
-as explained at https://doc.ez.no/display/EZP/Installing+eZ+Publish+Legacy+on+top+of+eZ+Platform.
+eZ Platform aiming to be **100% content compatible** with eZ Publish 5.x, 4.x and 3.x *(introduced in 2002)*, meaning
+that content in those versions of the CMS can be upgraded using [online documentation](http://doc.ez.no/eZ-Publish/Upgrading)
+to eZ Platform.
```
Comment (lolautruche) [inline README.md]: I guess can go on the line above..

**Comment by lolautruche** [comment]
+1

File: GETTING_STARTED.md (hunk 2 above)
Comment (andrerom) [inline GETTING_STARTED.md]: It is a todo, link is missing since there afaik isn't any yet ;)

File: README.md (hunk 3 above)
Comment (andrerom) [inline README.md]: On the intro line which kind of covers the same, or are you referring to not breaking up the line in the raw version here to avoid hitting the line length limit? :)

File: doc/nginx/etc/nginx/sites-available/mysite.com (hunk 1 above)
Comment (bdunogier) [inline doc/nginx/etc/nginx/sites-available/mysite.com]: Yes, it depends on the IO BinaryData handler. DFS is the only one that can take advantage of this right now.

We could say something like "Cluster/streamed files rewrite rules. Use only with DFS as a binary data handler".

**Comment by bdunogier** [comment]
+1

File: doc/nginx/etc/nginx/sites-available/mysite.com (hunk 1 above)
Comment (andrerom) [inline doc/nginx/etc/nginx/sites-available/mysite.com]: fixed in a7966ae
